Cat2Cat/main.m

```diff
@@ -207,10 +218,34 @@ int main(int argc, const char * argv[])
         [assetCatalogPaths addObjectsFromArray:filesMatchingPattern(url.path)];
     }
     parameters.assetCatalogPaths = assetCatalogPaths;
-    
+
+    parameters.methodNamePrefix = methodNamePrefix;
+
     NSURL *outputUrl = [NSURL URLWithString:[outputDirectory stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding]
                               relativeToURL:basePathUrl];
     parameters.outputDirectory = outputUrl.path;
     
     return parameters;
 }
+
+static BOOL validateMethodNamePrefix(NSString *methodNamePrefix, NSError **error)
+{
+    if ([methodNamePrefix length] < 2) {
+        *error = [NSError errorWithDomain:@"Cat2Cat" code:-1 userInfo:@{ NSLocalizedDescriptionKey : @"Method name prefix must be at least 2 characters." }];
+    } else {
+        NSMutableCharacterSet *charSet = [[NSCharacterSet letterCharacterSet] mutableCopy];
+        [charSet addCharactersInString:@"_"];
```

Comment:
I think numbers might be allowable, but not in the first character, but I'm not sure—but even if they are, I'm not sure it's worth allowing them here.